### PR TITLE
Change note about okta/azuread scim being avail for enterprise

### PR DESCRIPTION
**Pull request by butterkitty**
This is an automatically enabled feature now for enterprise customers and we no longer have to enable this manually, unless it's a custom provider

### CODE REVIEW

**Comment by butterkitty** [comment]
Making sure this change is ready for prime time. Asked the foundation team in their campfire for their input.

https://3.basecamp.com/3453178/buckets/23328660/chats/3993480919@6184047250

File: pages/integrations/sso/okta.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Customers on the Buildkite [Enterprise](https://buildkite.com/pricing) plan can
 Using the SCIM provisioning settings in Okta, Buildkite customers on the [Enterprise](https://buildkite.com/pricing) plan can automatically remove user accounts from your Buildkite organization. In Okta this feature is called 'Deactivating' a user. You need an enabled Okta SSO Provider before you can set up SCIM.
 
 >📘
-> User deprovisioning is an Enterprise only feature that needs to be enabled. Please contact support at support@buildkite.com to have this enabled.
+> User deprovisioning is an Enterprise only feature and is enabled automatically for non custom providers. If you are using a custom provider as an Enterprise customer please contact support@buildite.com to have this feature enabled.
```
Comment (JuanitoFatas) [inline pages/integrations/sso/okta.md:27]: How about re-arrange some words and linkify the custom provider? :)

```suggestion
> User deprovisioning is an Enterprise only feature and automatically enabled. If you are using a [custom provider](/docs/integrations/sso/custom-saml) as an Enterprise customer, please contact support@buildite.com to have this feature enabled.
```


[AZure AD integration](https://buildkite.com/docs/integrations/sso/azure-ad) would love to be included with this update too 🙌🏻 
https://github.com/buildkite/docs/blob/9975b995a6d502af2735e96eb064304d68de8846/pages/integrations/sso/azure_ad.md#configuration-instructions

Comment (butterkitty) [reply pages/integrations/sso/okta.md]: I love these changes. They look really sharp, thanks!

I didn't realize it was also on Azure AD, I'll add that one to the docket and update the PR! :slightly_smiling_face:

**Comment by butterkitty** [comment]
Added change to AzureAD page as per @JuanitoFatas recommendation

**Review by JuanitoFatas — APPROVED** [review]
Looks good to me 💚 Thank you!!

**Review by JuanitoFatas — APPROVED** [review]
Looks good to me 💚 Thank you!!

**Comment by JuanitoFatas** [comment]
Lovely 💝

**Pull request merged**